app/services/post_parser.py

```diff
@@ -28,3 +28,44 @@ def extract_contact_info(text):
     matches_email.extend(re.findall(email_pattern, text))
     contact = {"mobile": matches_phone, "email": matches_email}
     return contact
+
+
+def seperate_posts_from_whatsapp_bulk_text(bulk):
+    """
+    Add WhatApp text bulk.
+
+    Parameters
+    ----------
+    bulk: string
+
+    Returns
```

Comment:
Please return pairs instead of only the message: 
1. Datetime and contact, e.g. `[8:17 pm, 17/06/2024] Another Name`
2. The post (message).

Reply:
An addition functionality:
We need an app called "parser" that takes an input text file path & output file path as a parameters, loads the file content, pass it to `seperate_posts_from_whatsapp_bulk_text_to_json` and save its return it in the output file